src/components/SchemaResourcePage/SchemaResourceEdit.tsx

```diff
@@ -218,262 +219,276 @@ export function SchemaResourceEdit({
-                      ) : (
-                        <div className="flex justify-between">
-                          <h2 className="text-dark-gray font-bold mr-3 text-xl flex items-center space-x-2">
-                            {supportsField("icon") && <Icon name={icon} />}
-                            <span>{name}</span>
-                          </h2>
-                          {!!source && (
-                            <div className="px-2">
-                              <a href={`${source}`}>Config source</a>
-                            </div>
-                          )}
-                        </div>
-                      )}
-                      {supportsField("namespace") && (
-                        <div className="mt-4">
-                          <Controller
-                            control={control}
-                            name="namespace"
-                            render={({ field: { onChange, value } }) => {
-                              return (
-                                <TextInput
-                                  label="Namespace"
-                                  id="namespace"
-                                  disabled={disabled || !!id}
-                                  className="w-full"
-                                  value={value || ""}
-                                  onChange={onChange}
-                                />
-                              );
-                            }}
-                          />
-                        </div>
-                      )}
-                      {supportsField("labels") && (
-                        <div className="py-4">
-                          <Controller
-                            control={control}
-                            name="labels"
-                            render={({ field: { onChange, value } }) => {
-                              return (
-                                <div className="h-[100px]">
-                                  <label className="block text-sm font-bold text-gray-700">
-                                    Labels
-                                  </label>
-                                  <CodeEditor
-                                    key={labelsKeyRef.current}
-                                    readOnly={!!source || disabled || !edit}
-                                    value={
-                                      typeof values.labels === "object"
-                                        ? JSON.stringify(values.labels, null, 2)
-                                        : undefined
-                                    }
-                                    onChange={(val) => {
-                                      setValueOnChange("labels", val);
-                                    }}
-                                    language="json"
+    <>
+      <Head
+        prefix={`Settings ${resourceName} - ${
+          subNav.find((nav) => nav.value === activeTab)?.label
```

Comment:
Don't use label here, use `name` prop, which is the name displayed in the name field

Reply:
Done @mainawycliffe, please check again.